backend/fastrtc/utils.py

```diff
@@ -319,17 +320,15 @@ def audio_to_file(audio: tuple[int, NDArray[np.int16 | np.float32]]) -> str:
 
 
 def audio_to_float32(
-    audio: tuple[int, NDArray[np.int16 | np.float32]],
+    audio: NDArray[np.int16 | np.float32] | tuple[int, NDArray[np.int16 | np.float32]],
 ) -> NDArray[np.float32]:
     """
     Convert an audio tuple containing sample rate (int16) and numpy array data to float32.
 
     Parameters
     ----------
-    audio : tuple[int, np.ndarray]
-        A tuple containing:
-            - sample_rate (int): The audio sample rate in Hz
-            - data (np.ndarray): The audio data as a numpy array
+    audio : np.ndarray
```

Comment:
I've chosen not to document the legacy format so that people don't use it, but maybe it's confusing.

Reply:
I think it's ok. I'll remove the tuples api in a couple of weeks to give people a chance.